.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Thank you for reporting an issue with **AlexaPi**.
+
+Make sure you are running the latest version of **AlexaPi**, and look through [known issues](https://github.com/alexa-pi/AlexaPi/issues?utf8=%E2%9C%93&q=) before submitting.
+
+Our wiki may be found [here](https://github.com/alexa-pi/AlexaPi/wiki). Please spend a bit of time to review this before filing an issue.
+
+In order to help troubleshooting the issue, be sure to include the following information:
+
+- The OS (including version) where you are running **AlexaPi**.
+- The hardware platform and model you are running on.
+- Debug information as outlined in the [wiki](https://github.com/alexa-pi/AlexaPi/wiki/Debugging).
```

Comment:
Move this then up to the wiki info and link like [Debugging on wiki] or something like that - it links to the section, not the wiki.